packages/editor-ui/src/components/CredentialEdit/CredentialEdit.vue

```diff
@@ -770,15 +747,50 @@ async function saveCredential(): Promise<ICredentialsResponse | null> {
 	return credential;
 }
 
+const createToastMessagingForNewCredentials = (
+	credentialDetails: ICredentialsDecrypted,
+	project?: Project | null,
+) => {
+	let toastTitle = i18n.baseText('credentials.create.personal.toast.title');
+	let toastText = '';
+
+	if (!credentialDetails.sharedWithProjects) {
+		toastText = i18n.baseText('credentials.create.personal.toast.text');
+	}
+
+	if (projectsStore.currentProject) {
+		toastTitle = i18n.baseText('credentials.create.project.toast.title', {
+			interpolate: { projectName: project?.name ?? '' },
+		});
+
+		toastText = i18n.baseText('credentials.create.project.toast.text', {
+			interpolate: { projectName: project?.name ?? '' },
+		});
+	}
+
+	return {
+		title: toastTitle,
+		message: toastText,
+	};
+};
```

Comment:
nit: prefer function name(args) {} in top-level because they are hoisted even for other top-level variables.

```suggestion
function createToastMessagingForNewCredentials(
	credentialDetails: ICredentialsDecrypted,
	project?: Project | null,
) {
	let toastTitle = i18n.baseText('credentials.create.personal.toast.title');
	let toastText = '';
	if (!credentialDetails.sharedWithProjects) {
		toastText = i18n.baseText('credentials.create.personal.toast.text');
	}
	if (projectsStore.currentProject) {
		toastTitle = i18n.baseText('credentials.create.project.toast.title', {
			interpolate: { projectName: project?.name ?? '' },
		});
		toastText = i18n.baseText('credentials.create.project.toast.text', {
			interpolate: { projectName: project?.name ?? '' },
		});
	}
	return {
		title: toastTitle,
		message: toastText,
	};
}
```